Gem/Code/Source/XRRPIExampleComponent.cpp

```diff
@@ -37,13 +39,18 @@ namespace AtomSampleViewer
 
     void XRRPIExampleComponent::Activate()
     {
+        if (m_xrSystem = AZ::RPI::RPISystemInterface::Get()->GetXRSystem();
```

Comment:
Shouldn't the example illustrate the safest code logic and also check that the RPISystemInterface is not nullptr as well?

Since this is an example that is seen by new users, we don't want to show code that could potentially crash due to a nullptr dereference.

Reply:
I think the `RPISystemInterface` isn't an optional interface, at least not in Atom Sample Viewer.  It's always going to be there because it's fundamental to Atom.